src/shared/components/FavouriteButton/styles.scss

```diff
@@ -14,3 +14,18 @@
 .icon:hover {
   opacity: 0.6;
 }
+
+.anim {
```

Comment:
Would be awesome to name this animation better by e.g. what it does.

Reply:
Like you did with `showInFromNone`

Reply:
Done